client/js/components/chart.jsx

```diff
@@ -125,16 +151,19 @@ var Chart = React.createClass({
             }
         }
       });
+    },
+    renderVisualization: function(chartType, csvData, activeHeader, schemaName) {
+      if (chartType === "histogram") {
+        Chart.renderHistogramChart(csvData, activeHeader);
+      } else if (chartType === "timeSeries") {
+        Chart.renderTimeSeriesChart(csvData, activeHeader);
+      } else if (chartType === "scatterPlot") {
+        Chart.renderScatterplotChart(csvData, schemaName, activeHeader, 3);
```

Comment:
Scatterplots require two data points. Currently we only have ui artifacts to select one data point so 3 is the default index of the second data point. I will later add the ability for the user to choose the index of the second data point.